integration_tests/src/main/python/cache_test.py

```diff
@@ -91,11 +91,12 @@ def do_join(spark):
 @pytest.mark.parametrize('data_gen', all_gen, ids=idfn)
 @pytest.mark.parametrize('enable_vectorized_conf', enable_vectorized_confs, ids=idfn)
 @ignore_order
+@datagen_overrides(seed=0)
```

Comment:
Why no reason here?

Reply:
oh whoops, let me add that.. not intended

Reply:
I am re-running CI with these removed. I can't repro these locally.